spec/fixtures/generate_crt_and_Key_for_1028_keylength.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+mkdir ca && cd ca
+if [ $? -ne 0 ]; then
+   exit 1
+fi
+
+# 1. Creating a private key  
+openssl genrsa -out key_for_weak_key_cert.pem 1024
+
+# 2. Creating a certificate request 
+openssl req -verbose -new -key key_for_weak_key_cert.pem -out ca.csr  -subj "/C=US/ST=WA/L=Seattle/O=SUSE/OU=CaaSP/CN=1024 key length"
+
+# 3. Generating an openssl environment in the working directory (where the files are located at with SUSE OS)
+mkdir -p ./demoCA/newcerts; touch ./demoCA/index.txt; touch ./demoCA/index.txt.attr ;echo 01 > ./demoCA/serial
+
+# 4. Signing the certificate Request
+openssl ca -batch -extensions v3_ca -out weak_key_cert.pem -keyfile key_for_weak_key_cert.pem -selfsign -md sha256 -days 10000 -infiles ca.csr 
```

Comment:
Line is over the recommended line length; please go to another line by ending with a '\' to split it up to another line.